src/drm.hpp

```diff
@@ -68,6 +68,9 @@ struct connector {
 	uint32_t possible_crtcs;
 	std::map<std::string, const drmModePropertyRes *> props;
 	std::map<std::string, uint64_t> initial_prop_values;
+
+	char *make;
+	char *model;
```

Comment:
Where do these get initialized?

Reply:
Zero-initialized in `refresh_state`, with

```c
struct connector conn = { .id = conn_id };
```

Populated in `parse_edid` when the connector status switches to "connected".